cmd/common-main.go

```diff
@@ -157,15 +157,19 @@ func minioConfigToConsoleFeatures() {
 		}
 	}
 	// Enable if prometheus URL is set.
-	if value := env.Get("MINIO_PROMETHEUS_URL", ""); value != "" {
+	if value := env.Get(config.EnvMinIOPrometheusURL, ""); value != "" {
 		os.Setenv("CONSOLE_PROMETHEUS_URL", value)
```

Comment:
Shouldn't be using the constant `config.EnvMinIOPrometheusURL` ?

Reply:
For Console we do not have the console ENVs listed, it `minio/console/pkg/constants` provide this it is perhaps easier to use it inside MinIO since MinIO uses console as a library.

// cc @bexsoft @pjuarezd but that's a future enhancement and we do not have to hold this PR to.

Reply:
okay, then it can be done later